TraceScreen.c

```diff
@@ -90,11 +90,22 @@ bool TraceScreen_forkTracer(TraceScreen* this) {
 
       char buffer[32] = {0};
       xSnprintf(buffer, sizeof(buffer), "%d", this->super.process->pid);
-      execlp("strace", "strace", "-T", "-tt", "-s", "512", "-p", buffer, NULL);
 
-      // Should never reach here, unless execlp fails ...
-      const char* message = "Could not execute 'strace'. Please make sure it is available in your $PATH.";
-      (void)! write(STDERR_FILENO, message, strlen(message));
+      #if defined(HTOP_FREEBSD) || defined(HTOP_OPENBSD) || defined(HTOP_NETBSD) || defined(HTOP_DRAGONFLYBSD) || defined(HTOP_SOLARIS)
+         execlp("truss", "truss", "-s", "512", "-p", buffer, NULL);
+
+         const char* message = "Could not execute 'truss'. Please make sure it is available in your $PATH.";
+         (void)! write(STDERR_FILENO, message, strlen(message));
```

Comment:
Split Solaris case and drop `-s 512`: unsupported according to [manual](https://docs.oracle.com/cd/E88353_01/html/E37839/truss-1.html).